src/utils/legacy.ts

```diff
@@ -3,4 +3,5 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-export const isLegacy = Number.parseInt(window.OC?.config.version.split('.')[0] ?? '32') < 32
+const version = window.OC?.config?.version || '32.0.0'
+export const isLegacy = Number.parseInt(version?.split('.')[0] ?? '32') < 32
```

Comment:
But in case we want to be more graceful then keep the above and then we do not need the fallback here

```suggestion
export const isLegacy = Number.parseInt(version?.split('.')[0]!) < 32
```

Reply:
And now it complains again :grin: 
`error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.`

Adding the fallback back